apps/judicial-system/web/src/utils/formHelper.ts

```diff
@@ -317,3 +321,22 @@ export const findFirstInvalidStep = (steps: string[], theCase: Case) => {
 
   return key
 }
+
+export const isTrafficViolationIndictmentCount = (
+  policeCaseNumber?: string | null,
+  indictmentSubtypes?: IndictmentSubtypeMap,
+) => {
+  if (!policeCaseNumber || !indictmentSubtypes) {
+    return false
+  }
+
+  if (
+    indictmentSubtypes[policeCaseNumber].length === 1 &&
+    indictmentSubtypes[policeCaseNumber][0] ===
```

Comment:
Is it guaranteed that the subtype `TRAFFIC_VIOLATION` will always be the first instance of the array?

Reply:
AH we check if the array is only has exactly one element 👍 🙈